dbus/methods.go

```diff
@@ -204,7 +205,8 @@ func (c *Conn) KillUnit(name string, signal int32) {
 // KillUnitContext takes the unit name and a UNIX signal number to send.
 // All of the unit's processes are killed.
 func (c *Conn) KillUnitContext(ctx context.Context, name string, signal int32) {
-	c.KillUnitWithTarget(ctx, name, All, signal)
+	// No way to return an error.
+	_ = c.KillUnitWithTarget(ctx, name, All, signal)
```

Comment:
I agree with the change but I wonder if we should mark this function as deprecated and let people use KillUnitWithTarget given this here just eats the error silently.

Reply:
Was thinking the same thing. Amended.